README.md

```diff
@@ -67,7 +67,7 @@ These sections show how to use the SDK to perform API management functions. Befo
 10. [Impersonate](#impersonate)
 11. [Audit](#audit)
 12. [Embedded Links](#embedded-links)
-13. [Manage ReBAC Authz](#manage-rebac-authz)
+13. [Manage FGA Authz](#manage-fga-authz)
```

Comment:
Seems redundant to say Manage FGA Authz since that means Manage Fine-grained Authorization Authorization. Maybe just Manage FGA?

Reply:
I think it should be:
```
Manage FGA (Fine-grained Authorization)
```